### PR TITLE
Add Puri.sm.

### PR DESCRIPTION
**Pull request by lamby**

### CODE REVIEW

**Comment by Mayeu** [comment]
Hey :wave:

Thank you for your pull requests, it's really awesome that you considered improving this list :pray:

I did not expect to see hardware company so soon in the list, so I totally ignored this case until now :sweat_smile:

My logic with this list is that the money-making part of the business should be accessible on an open licence, in the case of a hardware company that would mean the hardware design/schematic is openly accessible, distributable, and modifiable. But as far as I know you can't access the schematic of my laptop (I am the proud owner a Librem 13v2 :) ).

An example of a hardware company that open source their hardware part would be Arduino, allowing other to rebuild the whole system from scratch (with the usual caveat in hardware where most elements of the board are not libre, but the design itself is).

What do you think @lamby? Do you know if I can find the schematic of Purism device somewhere? (Takin a look at their website right now, but I fail to find this :( )

**Comment by lamby** [comment]
@Mayeu

Purism do not currently ship schematics of their (varied) hardware offerings but there is no thematic or philosophical objection to why they would object to such a thing — I am 99% certain that they either do not exist (due to the OEM relationships preventing such a thing) or they would not be very useful if they did exist. I trust you have seen their development around the "Librem 5" hardware too where, even if actual actual schematics are not currently available, it is surely entirely fair to say that this is a free software oriented organisation and development model worthy of emulation by others.

However, I still believe Purism is appropriate here as the PureOS distribution that runs on the hardware is 100% free software, even to the point of being approved by the Free Software Foundation. Surely that is noteworthy in itself for inclusion here?

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ Please help me grow this list :) You could:
 - :boot: [Monica](https://www.monicahq.com/), [on GitHub](https://github.com/monicahq/monica): CRM for personal relationships.
 - :boot: [NextCloud](https://nextcloud.com/), [on GitHub](https://github.com/nextcloud): self-hosted personal cloud solution (file syncing, agenda, contact, etc.)
 - :boot: [Open Whisper Systems](https://www.signal.org/), [on GitHub](https://github.com/signalapp): creating Signal, a fast, simple, and secure communication app.
+- :moneybag: [Puri.sm](https://puri.sm/), [Cogs](https://code.puri.sm/), [on GitHub](https://github.com/purism): High-quality laptops that protect your freedom and privacy.
```
Comment (Mayeu) [inline README.md]: To follow the pattern used in the other line, can you:
- replace `[Cogs](https://code.puri.sm/)` by `[on their Cogs](https://code.puri.sm/)`
- add the funding amount with a link to crunchbase just after the link? e.g.: `[$5.6M](https://www.crunchbase.com/organization/purism)`

Thank you :pray:

**Comment by Mayeu** [comment]
@lamby Don't get me wrong, their work is noteworthy, and english not being my mothertongue, I hope that my answer did not seems harch :smiley: 

I am not against integrating Puri.sm in the list, what I want to be sure is that we are conscious that this will define the direction of the list for organisation creating software and hardware.

I am other fine with the inclusion of Puri.sm in the list :+1:

**Comment by Mayeu** [comment]
As said in #19, we will add Puri.sm as is, and we will see the result of #27 to possibly separate hardware company.

@lamby: I'll merge as soon as you finish the requested changes :+1:

**Comment by Mayeu** [comment]
Great :+1:

Merging the PR :)

**Pull request merged**